### PR TITLE
Add MayaData Hacktoberfest Opportunity

### LINKED ISSUE #684: MayaData Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : MayaData

Q. Where have you seen the announcement (URL)?  
A. : https://blog.mayadata.io/celebrate-hacktoberfest-2020-open-source-with-mayadata

Q. What kind of swag (clothing, stickers, ...)?  
A. : t-shirt

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by rmorabia**
This closes #684.

- [X] I've checked that this isn't a new swag opportunity proposal.
- [X] I've checked that this isn't a duplicate pull request.

This updates the entry from last year. If we need to add a new one instead, let me know. Happy to change.

### CODE REVIEW

**Comment by rmorabia** [comment]
Going to wait to get confirmation that I did things right here, then if this gets approved, happy to take on the other Hacktoberfest issues that were created. :)

File: data.json [hunk 1]
```diff
@@ -335,11 +335,11 @@
...
-    "description": "Get swag for contributing in general. At the end of the month-long event, the individual who makes the most significant contributions will receive a brand new laptop.",
-    "reference": "https://blog.mayadata.io/mayadata-celebrates-hacktoberfest-2019",
-    "image": "https://blog.mayadata.io/hs-fs/hubfs/Hacktoberfest-11-1.png",
-    "dateAdded": "2019-10-02T16:48:14.000Z",
-    "tags": ["clothing", "device", "hacktoberfest", "stickers", "expired"]
+    "description": "Get a t-shirt for contributing a pull request to LitmusChaos/OpenEBS. There is also exclusive swag and prizes for the top weekly contributors.",
+    "reference": "https://blog.mayadata.io/celebrate-hacktoberfest-2020-open-source-with-mayadata",
+    "image": "https://blog.mayadata.io/hs-fs/hubfs/Hacktoberfest%20desktop%20version.png?width=3001&name=Hacktoberfest%20desktop%20version.png",
+    "dateAdded": "2020-09-30T03:19:14.000Z",
+    "tags": ["clothing", "device", "hacktoberfest", "stickers"]
```
Comment (vikaspotluri123) [inline data.json:342]:
```suggestion
    "tags": ["clothing", "hacktoberfest", "stickers"]
```

**Review by vikaspotluri123 — COMMENTED** [review]
Everything else looks good to me!

**Review by aslafy-z — APPROVED** [review]
LGTM

**Pull request merged**